freemocap/core_processes/process_motion_capture_videos/process_session_folder.py

```diff
@@ -130,12 +143,15 @@ def process_session_folder(
         medipipe_reprojection_error_data_npy_path=s.recording_info_model.mediapipe_reprojection_error_data_npy_file_path,
     )
 
-    logger.info("Breaking up big `npy` into smaller bits and converting to `csv`...")
-    # break up big NPY and save out csv's
-    convert_mediapipe_npy_to_csv(
-        mediapipe_3d_frame_trackedPoint_xyz=skel3d_frame_marker_xyz,
-        output_data_folder_path=s.recording_info_model.output_data_folder_path,
-    )
+    # throw saving function into an if statement, because if we have a single camera, we already saved the data
+    if mediapipe_pose_world_data.shape[0] > 1:
```

Comment:
This is wrong - You should only need to have this `if` statement in *one* location in the code, and everything should be back to "normal" after it is passed. 

You need to put your single cam flattened skelly data into the `skel3d_frame_marker_xyz` variable in the `if` block above, and SKIP the saving step within that block. 

That way, when the code gets to this point, it will save the data you want